command/v6/service_command.go

```diff
@@ -183,6 +183,7 @@ func (cmd ServiceCommand) displayUserProvidedServiceInstanceSummary(serviceInsta
 	table := [][]string{
 		{cmd.UI.TranslateText("name:"), serviceInstanceSummary.Name},
 		{cmd.UI.TranslateText("service:"), cmd.UI.TranslateText("user-provided")},
+		{cmd.UI.TranslateText("route service url:"), serviceInstanceSummary.RouteServiceURL},
```

Comment:
@jeffleecoffee : Would be nice to have a unit test for this in `service_command_test.go` too. Thanks

Reply:
Good idea, I've made some changes to only print routes when the service has it and also added an extra unit test for service_command_test.